dotCMS/src/integration-test/java/com/dotcms/enterprise/license/LicenseManagerTest.java

```diff
@@ -65,7 +66,7 @@ public void test_checkServerDuplicity_dups() throws Exception {
                     .loadResult();
             added = true;
 
-            assertNotEquals(LicenseLevel.COMMUNITY.level, LicenseManager.getInstance().license.level);
+          //  assertNotEquals(LicenseLevel.COMMUNITY.level, LicenseManager.getInstance().license.level);
```

Comment:
why is this commented? Are you sure the license used is community?

Reply:
rolling all this back, not my changes. They're probably from the focal point branch